datafusion/core/tests/sql/expr.rs

```diff
@@ -448,8 +448,10 @@ async fn test_substring_expr() -> Result<()> {
     Ok(())
 }
 
+/// Test string expressions test split into two batches
```

Comment:
I am not concerned -- your comment is good enough

What we have found is that this large stack frame thing only affects debug builds (I think because the rust compiler isn't reusing stack space so debugger output is better) -- release builds have much smaller stack requirements

In this case could probably make individual test functions for each expression rather than a few functions that each test more than one function

Reply:
Thanks for the info, good to know this is specific to debug builds!